packages/cubejs-backend-shared/src/logger.ts

```diff
@@ -0,0 +1,132 @@
+import SqlString from 'sqlstring';
```

Comment:
As I understand you moved the logger to `@cubejs-backend/shared` to create a new instance of it, but the logger should be re-used from the server-core, instead of creating a new one in each place where it should be used.

Reply:
Ah. Fell into the logging pattern we use where I work.